### DISCUSSION TITLE
What should our webapp be called?

### DISCUSSION THREAD

**Discussion by nelsonmestevao** [joinum/parzival#75] [Q&A]
At the moment, our code name for the web platform for JOIN is called `parzival`. @feliciofilipe can explain the reason behind that name better than anyone. 

What other names could we choose? I think we could come up with some rule that would make sense to the 3 students associations working on the open source projects but we should follow simple rules like:

- The names should be relatively short (I think the current name meets this requirement)
- The names should be easy to remember/say (I think the current name does not meets this requirement since I have seen people write parcival [checkout heroku apps] and listen to people saying "parval" :D)
- The names should make sense in the scope of the project or the general goals/rules

**Comment by feliciofilipe** [comment]
JOIN is an annual Job Shop aimed at students in the field of computing.

The platform revolves around improving the experience in the event and better accomplish its goals. 

With that in mind, inside the pursuing (internship, job offer, etc)  mindset, Parzival was an arthurian hero that venture into a long quest for the Holy Grail (by analogy, any elusive object or goal of great significance may be perceived as a holy grail by those seeking it) following his initial failure to achieve it.

Also it is a nod to the software stack we are using (Elixir, Phoenix), since the holy grail and the arthurian tales are often associated with the pseudo-science of alchemy.

The story of the knights of the round table fits perfectly with the context/concept of three associations (four if we count the Informatics Department) joining forces: individuals with different backgrounds and motivation that band together in the name of achieving something greater than themselves.

Those were a few of the reasons behind my choice. I suggest that all of our repos here have names from knights of the round table and/or people from the arthurian tales:

https://en.wikipedia.org/wiki/Knights_of_the_Round_Table

Here is some examples:

- Lancelot
- Galahad
- Arthur
- Merlin

I think our core values such as team work, passion, sacrifice in the name of others and passing of these values amongst generations of students' representitives really fits into this motif.

**Reply by nelsonmestevao** [reply]
I think that list has easier names to say and remember like:

- [Cador](https://en.wikipedia.org/wiki/Cador)
- [Gareth](https://en.wikipedia.org/wiki/Gareth)
- [Kay](https://en.wikipedia.org/wiki/Sir_Kay)
- [Tristan](https://en.wikipedia.org/wiki/Tristan)

Merlin is pretty cool too.

**Reply by feliciofilipe** [reply]
For the particular details of his story, I think Parzival is the best fit for this platform.

The ones listed are easier, I agree. I understand the point you have made on having a easy being important, but I disagree that parzival is a hard name to remember or say. 

When we look to other repos in one of the associations' github, they are just as "hard" (e.g. safira, moonstone, atomic).

With that being said, the name of the repo is far from a hill that I would die on. Whatever the team decides, I fully support. :muscle:

**Reply by nelsonmestevao** [reply]
I think one night you should do a slideshow presentation telling everyone a full detail version of the Parzival story 😍
